api-examples/transformer_seq2seq_response.py

```diff
@@ -6,7 +6,8 @@
 import glob
 from argparse import ArgumentParser
 import baseline
-from transformer_utils import TiedEmbeddingsSeq2SeqModel, find_latest_checkpoint
+from transformer_utils import find_latest_checkpoint
```

Comment:
I thought we were getting rid of one of these

Reply:
done